signing_clients/apps.py

```diff
@@ -324,18 +322,20 @@ def signature(self):
         # section signatures
         return self.signatures.header + b"\n"
 
-    def make_signed(self, signature, outpath=None, sigpath=None):
-        outpath = outpath or self.outpath
+    def make_signed(self, signed_manifest, signature, outpath, sigpath):
         if not outpath:
             raise IOError("No output file specified")
```

Comment:
Since `outpath` is now a required argument this check should be irrelevant I think? the `os.path.exists` check below should take care of the rest.